### PR TITLE
Campaign and Canvas Revamp

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [x] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the chart below to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTtzHpcihaXTTYD85LoKIvYBvpCQFLr8n0BDKRDRAMEz_DnZdHJJINKL24r4JXkRUui24pl_DVxbu2T/pub)
- [Technical Documentation Style Guide](https://docs.google.com/document/d/e/2PACX-1vTtzHpcihaXTTYD85LoKIvYBvpCQFLr8n0BDKRDRAMEz_DnZdHJJINKL24r4JXkRUui24pl_DVxbu2T/pub#h.wstt3flbts5k)
- [Styling Test Page](https://github.com/braze-inc/braze-docs/blob/develop/_docs/_home/styling_test_page.md)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then `Docs`, this can take up to 5 minutes to load. A `502` error means the deployment is still loading, refresh the page, and try again.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@Timothy-Kim</td>
    <td>Application Infrastructure<br>Data Infrastructure</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Intelligence<br>Product Partnerships<br>SMS<br>Internal Tools</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Ingestion<br>Platforms and Channels<br>SMB</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Messaging and Automation<br>Dashboard Infrastructure<br>Email</td>
  </tr>
</table>
</details>

### CODE REVIEW

**Comment by bre-fitzgerald** [comment]
@cla-bot check

File: _docs/_user_guide/engagement_tools/campaigns/building_campaigns.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+nav_title: Building Campaigns
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/campaigns/building_campaigns.md:2]: @bre-fitzgerald Minor suggestion to align the "Building X" titles so it's either 
- "Building a Campaign" and "Building a Canvas" 
- "Building Campaigns" and "Building Canvases"

Comment (bre-fitzgerald) [reply _docs/_user_guide/engagement_tools/campaigns/building_campaigns.md:2]: Updated to "Building Campaigns" and "Building Canvases" to match verb form with Managing/Testing

**Review by lydia-xie — APPROVED** [review]
looks great to me!

File: _docs/_user_guide/engagement_tools/campaigns/faq.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ tool: Campaigns
```
Comment (KellieHawks) [inline _docs/_user_guide/engagement_tools/campaigns/faq.md:10]: Hey @bre, FAQ isn't showing up as the last thing on the left hand nav
Screen Shot 2021-09-29 at 12 53 30 PM

File: _docs/_user_guide/engagement_tools/canvas/get_started/the_basics.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-nav_title: The Basics
+nav_title: Canvas Basics
```
Comment (KellieHawks) [inline _docs/_user_guide/engagement_tools/canvas/get_started/the_basics.md:2]: Hey @bre, do you mind swapping out the "Find your strategy with the five W's of visualization" section here, for the one in the Campaign basics doc I made. That one has an updated list and is formatted correctly.

**Comment by KellieHawks** [comment]
minor nit, can you move experiment step above FB audience step and google audience step so the same type of steps are all grouped together in the right-hand nav
Screen Shot 2021-09-29 at 1 05 09 PM

**Review by KellieHawks — CHANGES_REQUESTED** [review]
Looks so great! Thank you for all the hard work on this. It looks so much better!! 😭 Just a couple of minor things I caught.

**Comment by bre-fitzgerald** [comment]
> minor nit, can you move experiment step above FB audience step and google audience step so the same type of steps are all grouped together in the right-hand nav

@KellieHawks 
We originally decided to group it as it is ordered in the UI. I don't mind the change, just want to double-check 
cc: @lydia-xie

**Comment by KellieHawks** [comment]
Ah gotcha. Wasn't sure the reason, but that makes lots of sense. We can leave it. :)

**Review by KellieHawks — APPROVED** [review]
💯💯💯

**Pull request merged**